### DISCUSSION TITLE
寻求帮助！我的资产被盗了！！

### DISCUSSION THREAD

**Discussion by boxiaoyizhan** [Chia-Network/chia-blockchain#9160] [Support]
hi
我的资产再前几天早上转到客户端。确定已收到。
然后一直无法连接钱包。后更新重新下载重新同步。
现在登录进去发现钱包的资产被转进去的当天下午被转走。
请帮帮我。

**Comment by loppefaaret** [comment]
it sounds like someone might have been able to either get access to your machine, or otherwise get a hold of your private key ? in that case, it is very unlikely that you are able to get your funds back!
